conf/cmi/core.entity_form_display.tpr_unit.tpr_unit.default.yml

```diff
@@ -415,4 +426,3 @@ hidden:
   field_study_field: true
   hide_description: true
   ontologyword_ids: true
-  show_www: true
```

Comment:
Is this change fine? I don't actually know what this field is or in which instances it's needed.